pkg/apis/admission/BUILD

```diff
@@ -0,0 +1,46 @@
+package(default_visibility = ["//visibility:public"])
```

Comment:
@whitlockjc For the future, please try to separate autogenerated changes into a separate commit - it's much easier to review then.